src/app.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+
+import logger from './utils/logger.utils.js';
+import healthRoutes from './routes/health.routes.js';
+import moviesRoutes from './routes/movies.routes.js';
+import swaggerDocs from './utils/swagger.js';
+
+const port = 3000;
+
+const app = express();
+
+app.use(express.json());
+
+app.listen(port, () => {
+	logger.info(`App listening on http://localhost:${port}`);
+
+	app.use('/movies', moviesRoutes);
```

Comment:
It's not necessary to place lines 17-27 inside this event handler. Here you need just print message to make sure that app is running. Move lines 17-27 below the middleware in line 12

Reply:
Done!